Find a functionally equivalent code:android.os.ParcelFileDescriptor.createReliablePipe:COMMENT
Method Modifier: public      static      
Comment:/**
 * Create two ParcelFileDescriptors structured as a data pipe. The first
 * ParcelFileDescriptor in the returned array is the read side; the second
 * is the write side.
 * <p>
 * The write end has the ability to deliver an error message through
 * {@link #closeWithError(String)} which can be handled by the read end
 * calling {@link #checkError()}, usually after detecting an EOF.
 * This can also be used to detect remote crashes.
 */

@@ -1,7 +1,7 @@
 {
     try {
-        final FileDescriptor[] comm = createCommSocketPair(false);
-        final FileDescriptor[] fds = Libcore.os.pipe();
+        final FileDescriptor[] comm = createCommSocketPair();
+        final FileDescriptor[] fds = Os.pipe();
         return new ParcelFileDescriptor[] { new ParcelFileDescriptor(fds[0], comm[0]), new ParcelFileDescriptor(fds[1], comm[1]) };
     } catch (ErrnoException e) {
         throw e.rethrowAsIOException();

